pkg/cli/admin/release/info.go

```diff
@@ -666,7 +666,7 @@ func calculateDiff(from, to *ReleaseInfo) (*ReleaseDiff, error) {
 		}
 		diff.ChangedManifests[name] = &ReleaseManifestDiff{
 			Filename: name,
-			From:     manifest,
+			To:       manifest,
```

Comment:
I'm surprised to see such a long standing and missed bug and wonder how this didn't cause any problems.